sefaria/model/place.py

```diff
@@ -46,8 +46,24 @@ def primary_name(self, lang=None):
 
     def secondary_names(self, lang=None):
         return self.name_group.secondary_titles(lang)
-
-    ###
+    
+    @classmethod 
+    def create_new_place(cls, en, he=None):
+        p = Place().load({'key': en})
+        if p:
+            return p
+        p = Place({'key': en})
+        p.name_group.add_title(en, 'en', True, True)
+        if he:
+            p.name_group.add_title(he, 'he', True, True)
+        p.city_to_coordinates(en)
+        p.save()
+        return p
+
+    def city_to_coordinates(self, city):
+        geolocator = Nominatim(user_agent='hello@sefaria.org')
```

Comment:
very good thinking out of the box! using Nominatim for finding the coords is great, what were they doing till now? does that data not come in in other ways?